packages/types/src/network.ts

```diff
@@ -11,4 +11,5 @@ export type InvitationPair = {
 export type InvitationData = {
   pairs: InvitationPair[]
   psk: string
+  ownerOrbitDbIdentity: string
```

Comment:
Don't forget to adjust mobiles' containers as well as desktops'. They both share this interface

Reply:
I was thinking to do the mobile work in a separate PR. What do you think?

Reply:
So for this specific case I think I'd rather be good to do it immediately as it's a matter of unifying interfaces. That means once we merged this PR, mobile stops working until we fix it and it's not desirable.